server.js

```diff
@@ -62,12 +62,29 @@ function broadcastPlayers() {
 
 
 function updatePlayerPosition(player) {
+  if (player.lastDeath !== null) {
+    const currentDate = new Date()
+    if (player.lastDeath > currentDate) {
+      return
+    } else {
+      io.to(player.id).emit('respawn-end')
+      player.lastDeath = null
+    }
+  }
+
   if (player.shield < 0) {
-    player.x = Math.floor(Math.random() * 1920) + 1;
-    player.y = Math.floor(Math.random() * 1080) + 1;
+    player.x = 50;
+    player.y = 50;
```

Comment:
This above reads that we'll always spawn at 50,50, I think we should revert back to the 'random' spawn placement.

Reply:
Fixed https://github.com/LukeGarrigan/stream-royale/pull/2/commits/917f4ec823c5159802f384e81010e5fbe9689786